packages/pds-ember/.gitignore

```diff
@@ -18,6 +18,7 @@
 /npm-debug.log*
 /testem.log
 /yarn-error.log
+.storybook/preview-head.html
```

Comment:
generated file, root `.gitignore` seems to be ignored while `cwd` is `packages/pds-ember` (will address refactoring gitignore files in future PR)

Reply:
Good catch on that one